src/Cake.Common/Tools/DotNetCore/Restore/DotNetCoreRestorer.cs

```diff
@@ -120,6 +120,12 @@ private ProcessArgumentBuilder GetArguments(string root, DotNetCoreRestoreSettin
                 builder.Append("--no-dependencies");
             }
 
+            // Force restore
+            if (settings.Force)
+            {
+                builder.Append("--force");
```

Comment:
Sorry for the delay in feedback, could you please add an test for this in [src/Cake.Common.Tests/Unit/Tools/DotNetCore/Restore/DotNetCoreRestorerTests.cs](https://github.com/cake-build/cake/blob/develop/src/Cake.Common.Tests/Unit/Tools/DotNetCore/Restore/DotNetCoreRestorerTests.cs).